README.md

```diff
@@ -3,14 +3,22 @@
 
 ![Screenshot of PyCob Quickstart App](https://cdn.pycob.com/quickstart_screenshot.png)
 
-## Create web apps using just Python
+We created PyCob to let you quickly create functional apps using only python. Read more about the project goals here
+
 
+## Create web apps using just Python
+The app from the above screenshot can be started/created/is possible with just the following steps
 ### Install PyCob
 ```bash
+
+git clone https://github.com/pycob/pycob
+cd pycob
+# activate your venv however appropriate
```

Comment:
You can link out to a separate article walking through poetry, venv, and conda, when to use each.  

Conda is great when you are linking against your own custom c/cython code, otherwise at this point it's a bit overkill.